### DISCUSSION TITLE
How to write the Matrix element for the Majorana interaction?

### DISCUSSION THREAD

**Discussion by maksymovchynnikov** [FeynCalc/feyncalc#192] [Q&A]
Consider a Lagrangian

L = gphi *phi nubar_L C nu_L + h.c.,

where gphi is some real constant, phi is a real scalar field, nubar and nu are neutrino fields, and C is the charge conjugation operator. nubar C nu is the Majorana scalar bilinear. Could you please tell me how to write down the matrix element and evaluate the squared matrix element for the process phi -> nu nu in FeynCalc? Something like

Matr = gphi/2 SpinorUBar[pnu1,0].C.(1-GA5).SpinorV[pnu2,0]//Contract
MatrStar=ComplexConjugate[Matr]
MatrSquared= FermionSpinSum[Matr MatrStar]/.DiracTr->TR//Contract//Simplify  ?

If yes, then what symbol in FeynCalc represents the charge conjugation operator?

**Comment by vsht** [comment]
There is no charge conjugation operator in FeynCalc. You can avoid it by using Denner's prescription for writing down the vertices:
https://inspirehep.net/literature/32906

That one is implemented in FeynArts. So if you create a FeynRules model for your Lagrangian, export it to FeynArts and then generate the corresponding Feynman diagrams, calculating them with FeynCalc should work out of the box. This is what I use myself.

If you prefer to write everything by hand, then I'm afraid that I can't help you here. Most likely one could cook up something that would work as a C matrix, but personally I have no experience with that. Too error-prone for my taste.
